src/styles/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 0 16px;
+  box-sizing: border-box;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+  display: flex;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--main-text-color);
+}
+
+.card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0;
+  margin-bottom: 16px;
+  color: var(--secondary-text-color);
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 23px;
+}
+
+.card__reviews-number {
+  font-size: 10px;
+  margin: 0;
```

Comment:
you can just reset default margin for all `h` and `p` elements instead of adding this rule to an each element